files/es/learn/common_questions/thinking_before_coding/index.html

```diff
@@ -74,35 +74,35 @@ <h3 id="¿Qué_es_exactamente_lo_que_quiero_lograr">¿Qué es exactamente lo que
+<p>Encontrar pareja es un caso prioritario en el que tiene más sentido utilizar los recursos existentes, en vez de construir todo un nuevo sitio. ¿Por qué? Porque invertiríamos más tiempo construyendo y manteniendo el sitio web en lugar de realmente buscar una pareja. Dado que nuestro objetivo es lo que más importa, debemos gastar nuestra energía en aprovechar las herramientas ya existentes en lugar de empezar desde cero. Nuevamente, hay tantos servicios web ya disponibles para mostrar fotos que no vale la pena el esfuerzo para construir un nuevo sitio sólo para anunciar acerca de lo tiernos que son nuestros gatos.</p>
 
-<p>Los otros cinco objetivos están todos conectados con la música. Hay, por supuesto, muchos servicios web que podrían manejar estas metas, pero tiene sentido, en este caso, construir un sitio web propio. Tal sitio web es la manera de agregar todo lo que queremos publicar en un solo lugar (bueno para las metas 3, 5 y 6). En resumen, ya que estos objetivos giran en torno al mismo tema, tener todo en un solo lugar nos ayudará a alcanzar nuestras metas y facilitará a nuestros seguidores el conectarse con nosotros.</p>
+<p>Los otros cinco objetivos están todos conectados con la música. Hay, por supuesto, muchos servicios web que podrían manejar estas metas, pero en este caso tiene sentido construir un sitio web propio. Tal sitio web es la manera de incorporar todo lo que queremos publicar en un solo lugar (bueno para las metas 3, 5 y 6). En resumen, ya que estos objetivos giran en torno al mismo tema, tener todo en un solo lugar nos ayudará a alcanzar nuestras metas y facilitará a nuestros seguidores el conectarse con nosotros.</p>
 
 <p>¿Cómo puede un sitio web ayudarme a alcanzar mis metas? Al responder a eso, encontrarás la mejor manera de alcanzar dichas metas y te ahorrarás desperdiciar esfuerzo.</p>
 
 <h3 id="¿Qué_hay_que_hacer_y_en_qué_orden_para_alcanzar_mis_metas">¿Qué hay que hacer, y en qué orden, para alcanzar mis metas?</h3>
 
-<p>Ahora que sabes qué es lo que quieres lograr, es momento de volver los objetivos en pasos prácticos. Como una nota lateral, tus metas no están necesariamente petrificadas, ellas evolucionan con el tiempo incluso en el transcurso del proyecto, si te encuentras con obstáculos inesperados o simplemente cambias de opinión.</p>
+<p>Ahora que sabes qué es lo que quieres lograr, es momento de convertir los objetivos en pasos prácticos. Como una nota adicional, tus metas no están necesariamente escritas en piedra. Ellas evolucionarán con el tiempo aún durante el desarrollo del proyecto, especialmente si te encuentras con obstáculos inesperados o simplemente cambias de opinión.</p>
 
-<p>En lugar de una extensa explicación, volvamos a nuestro ejemplo con esta tabla.</p>
+<p>En vez de desarrollar una extensa explicación, volvamos a nuestro ejemplo con esta tabla.</p>
```

Comment:
* "En vez" es una frase coloquial

```suggestion
<p>En lugar de desarrollar una extensa explicación, volvamos a nuestro ejemplo con esta tabla.</p>
```